ledger/participant-state/kvutils/src/test/lib/scala/com/daml/ledger/participant/state/kvutils/ParticipantStateIntegrationSpecBase.scala

```diff
@@ -101,7 +101,7 @@ abstract class ParticipantStateIntegrationSpecBase(implementationName: String)(
       "provide an update" in participantState.use { ps =>
         val submissionId = newSubmissionId()
         for {
-          result <- ps.uploadPackages(submissionId, List(archives.head), sourceDescription).toScala
+          result <- ps.uploadPackages(submissionId, List(archive1), sourceDescription).toScala
```

Comment:
If it doesn't really matter what archive `archive1` is then let's call it e.g. `anArchive` and `archive2` `anotherArchive`.

Reply:
renamed